pulpcore/cli/common/generic.py

```diff
@@ -489,6 +489,61 @@ def null_callback(
     return value
 
 
+def version_list_callback(
```

Comment:
What makes this generic? How is it different from the one used in export?

Reply:
Is it different now?

Reply:
@mdellweg except for one line, which was removed by a different PR while my PR was active - works the same for this use case

Reply:
So one line is different, but we are still using the same callback?

Reply:
@mdellweg no, the line has nothing to do with my PR

let me put it this way (arbitary numbers of days):
- on day 1, I copy the function to generic.py
- on day 15, a PR (not mine) which adds a line to the original function gets merged to main (and the new line has no effect on my use case)
- on day 30, I delete the copy of the (older version) function and decide to use the main one, which is updated now, but it doesn't matter